test/e2e/tests/network-error.spec.js

```diff
@@ -8,6 +8,7 @@ describe('Gas API fallback', function () {
       .forGet(
         'https://gas-api.metaswap.codefi.network/networks/1/suggestedGasFees',
       )
+      .always()
```

Comment:
We call the [always](https://httptoolkit.github.io/mockttp/classes/RequestRuleBuilder.html#always) method to ensure this rule is run for the test duration.
In this test, the suggestedGasFees endpoint will be called multiple times. 
The first time would use this mock, then the times after ill use the mock from the mock-e2e file.
Calling `always` will insure all calls to the endpoint use this mock and it doesn't fallback to the mock-e2e file.